examples/tree_mini.r

```diff
@@ -23,5 +23,6 @@ test <- function() {
 
 f <- rir.compile(function() test())
 
```

Comment:
A comment here saying we need to run twice (at least at the moment) to avoid confusing our profiling with partial data may save a lot of time to other who eventually do not understand the rationale and remove one of the calls